frontend/src/scenes/pipeline/hogfunctions/PipelineHogFunctionConfiguration.tsx

```diff
@@ -138,9 +139,12 @@ export function PipelineHogFunctionConfiguration({
                                         )}
                                     </LemonField>
 
-                                    <div className="flex flex-col py-1 flex-1">
+                                    <div className="flex flex-col py-1 flex-1 justify-start">
                                         <span className="font-semibold">{configuration.name}</span>
                                     </div>
+
+                                    <HogFunctionStatusIndicator />
```

Comment:
This doesn't show up immediately after first enabling:

![2024-06-26 17 33 40](https://github.com/PostHog/posthog/assets/53387/8a251a38-7be1-4d03-9a31-5d668c5831bb)

Reply:
In the same vain: it doesn't disappear when disabling... and once disabled, I can't see the status history anymore.

Reply:
Ideally this information should also end up in the "status" column here:

<img width="1431" alt="image" src="https://github.com/PostHog/posthog/assets/53387/ed90bef2-4922-4b8f-a8e2-9ccd3b8ba125">

I instinctively tried clicking the tag to see the status history show up... only to realize it's a different one that says "active" not "healthy"

Reply:
Yeah so most of this is part of the follow up PR